packages/component-library/stories/PathMap.notes.md

```diff
@@ -0,0 +1,34 @@
+# Path Map Component
+
+## Standard
+
+The Standard story shows the standard usage of the Path Map Component for the CIVIC platform. Path Map includes the standard styling.
+
+These properties can be set in the standard story:
+
+- `getColor`
+- `getWidth`
+- `opacity`
+- `data`
+- `getPath`
+
+## Custom
+
+The Custom story shows all the possible properties that can be passed to the Path Map Component for customization.
+
+- `data` : expects an array of objects.
+- `getPath` : expects a function that returns an array of coordinates for each path. It is compatible with GeoJSON LineStrings.
+- `getColor` : expects a function or an array in `[r, g, b, [a]]` format, where `a` is a number between 0 and 255. If an array is provided, the color is applied to all objects. If a function is provided, it is called on each object to set the fill color.
+- `getWidth` : expects a function or a number. If a number is provided it is used as the radius for all objects. If a function is provided, it is called on each object to set the radius.
```

Comment:
"radius" should be changed to "width" twice in this section.

Reply:
Thanks for finding that! I fixed it in my lastest commit.